library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/utils/MiscUtilsTest.kt

```diff
@@ -114,15 +114,8 @@ internal class MiscUtilsTest {
         val fakeDisplayMetrics = DisplayMetrics().apply {
             density = fakeDensity
         }
-        val mockResources: Resources = mock {
-            whenever(it.configuration).thenReturn(fakeConfiguration)
-            whenever(it.displayMetrics).thenReturn(fakeDisplayMetrics)
-        }
-        val mockContext: Context = mock {
-            whenever(it.getSystemService(Context.WINDOW_SERVICE)).thenReturn(mockWindowManager)
-            whenever(it.resources).thenReturn(mockResources)
-            whenever(it.theme).thenReturn(mockTheme)
-        }
+        val mockResources: Resources = mockResources(fakeConfiguration, fakeDisplayMetrics)
```

Comment:
there are some repeated setup steps in each test - perhaps we can refactor some to  a beforeEach